src/db/build/configure-db.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Wait 60 seconds for SQL Server to start up by ensuring that
+# calling SQLCMD does not return an error code, which will ensure that sqlcmd is accessible
+# and that system and user databases return "0" which means all databases are in an "online" state
+# https://docs.microsoft.com/en-us/sql/relational-databases/system-catalog-views/sys-databases-transact-sql?view=sql-server-2017
+
+PATH=$PATH:/opt/mssql-tools/bin/
+
+TIMEOUT=60
+
+DBSTATUS=1
+EXIT_CODE=1
+i=0
+while [ $i -lt $TIMEOUT ]; do
+    i=$(($i+1))
+    DBSTATUS=$(sqlcmd -h -1 -t 1 -U sa -P $SA_PASSWORD -Q "SET NOCOUNT ON; Select SUM(state) from sys.databases" 2>/dev/null)
```

Comment:
Does `$SA_PASSWORD` come from an .env file that isn't in the repo, or is it misnamed? Looking at the `Dockerfile` shows me that it's `$MSSQL_SA_PASSWORD` (https://github.com/uwcirg/leaf/blob/master/src/db/build/Dockerfile#L15)

Reply:
Oh. Right. The `docker-compose.yml` above defines it. Silly me!